optimum/intel/openvino/modeling_decoder.py

```diff
@@ -232,25 +233,20 @@ def _from_transformers(
         onnx_config = onnx_config_constructor(model.config, use_past=use_cache)
 
         # TODO : create ModelPatcher to patch each architecture
-        if config.model_type == "bloom":
-            model.transformer._prepare_attn_mask = _prepare_attn_mask
-        elif config.model_type == "llama":
-            model.model._prepare_decoder_attention_mask = _prepare_decoder_attention_mask
-        elif config.model_type in {"blenderbot-small", "blenderbot", "opt", "pegasus", "bart"}:
-            model.model.decoder._prepare_decoder_attention_mask = _prepare_decoder_attention_mask
+        model = patch_decoder_attention_mask(model)
 
         # Export the model to the ONNX format
         export(model=model, config=onnx_config, output=save_dir_path / ONNX_WEIGHTS_NAME)
 
         return cls._from_pretrained(
             model_id=save_dir_path,
             config=config,
-            from_onnx=True,
+            from_onnx=not is_torch_model(model),
```

Comment:
It would make sense to have an OpenVINO model resulting from export for both pytorch / tensorflow wdyt ? Also could make sense to deprecate `from_onnx` in the future

Reply:
yes, openvino supports tf model conversion directly as well, but I can not find any testing examples in optimum-intel in support tf models to test this approach, that is why I keep it like in optimum used.
Do you have some models that I can take a look regarding tf support?